packages/datepicker/src/index.ts

```diff
@@ -1,4 +1,2 @@
-export { default } from './react'
-export * from './vars'
-export * from './js'
-export { default as css } from './css'
```

Comment:
I don't think we want to delete the css export. The file exists, was used and this pattern is used broadly. Any reason to delete?